config/config.go

```diff
@@ -9,25 +9,27 @@ import (
-	BindAddr                 string        `envconfig:"BIND_ADDR"`
-	Brokers                  []string      `envconfig:"KAFKA_ADDR"                     json:"-"`
-	DatasetAPIURL            string        `envconfig:"DATASET_API_URL"`
-	DatasetAPIAuthToken      string        `envconfig:"DATASET_API_AUTH_TOKEN"         json:"-"`
-	DimensionsExtractedTopic string        `envconfig:"DIMENSIONS_EXTRACTED_TOPIC"`
-	DimensionExtractorURL    string        `envconfig:"DIMENSION_EXTRACTOR_URL"`
-	EncryptionDisabled       bool          `envconfig:"ENCRYPTION_DISABLED"`
-	EventReporterTopic       string        `envconfig:"EVENT_REPORTER_TOPIC"`
-	GracefulShutdownTimeout  time.Duration `envconfig:"GRACEFUL_SHUTDOWN_TIMEOUT"`
-	InputFileAvailableGroup  string        `envconfig:"INPUT_FILE_AVAILABLE_GROUP"`
-	InputFileAvailableTopic  string        `envconfig:"INPUT_FILE_AVAILABLE_TOPIC"`
-	KafkaMaxBytes            string        `envconfig:"KAFKA_MAX_BYTES"`
-	MaxRetries               int           `envconfig:"REQUEST_MAX_RETRIES"`
-	VaultAddr                string        `envconfig:"VAULT_ADDR"`
-	VaultToken               string        `envconfig:"VAULT_TOKEN"                    json:"-"`
-	VaultPath                string        `envconfig:"VAULT_PATH"`
-	ServiceAuthToken         string        `envconfig:"SERVICE_AUTH_TOKEN"             json:"-"`
-	ZebedeeURL               string        `envconfig:"ZEBEDEE_URL"`
+	BindAddr                    string        `envconfig:"BIND_ADDR"`
+	Brokers                     []string      `envconfig:"KAFKA_ADDR"                     json:"-"`
+	DatasetAPIURL               string        `envconfig:"DATASET_API_URL"`
+	DimensionsExtractedTopic    string        `envconfig:"DIMENSIONS_EXTRACTED_TOPIC"`
+	DimensionExtractorURL       string        `envconfig:"DIMENSION_EXTRACTOR_URL"`
+	EncryptionDisabled          bool          `envconfig:"ENCRYPTION_DISABLED"`
+	EventReporterTopic          string        `envconfig:"EVENT_REPORTER_TOPIC"`
+	GracefulShutdownTimeout     time.Duration `envconfig:"GRACEFUL_SHUTDOWN_TIMEOUT"`
+	InputFileAvailableGroup     string        `envconfig:"INPUT_FILE_AVAILABLE_GROUP"`
+	InputFileAvailableTopic     string        `envconfig:"INPUT_FILE_AVAILABLE_TOPIC"`
+	KafkaMaxBytes               string        `envconfig:"KAFKA_MAX_BYTES"`
+	MaxRetries                  int           `envconfig:"REQUEST_MAX_RETRIES"`
+	VaultAddr                   string        `envconfig:"VAULT_ADDR"`
+	VaultToken                  string        `envconfig:"VAULT_TOKEN"                    json:"-"`
+	VaultPath                   string        `envconfig:"VAULT_PATH"`
+	ServiceAuthToken            string        `envconfig:"SERVICE_AUTH_TOKEN"             json:"-"`
+	ZebedeeURL                  string        `envconfig:"ZEBEDEE_URL"`
+	HealthCheckInterval         time.Duration `envconfig:"HEALTHCHECK_INTERVAL"`
+	HealthCheckRecoveryInterval time.Duration `envconfig:"HEALTHCHECK_RECOVERY_INTERVAL"`
```

Comment:
If this could change to `HealthCheckCriticalTimeout         time.Duration  'envconfig:"HEALTHCHECK_CRITICAL_TIMEOUT"'`

And update variable name where appropriate, thanks